creator-node/src/reqLimiter.js

```diff
@@ -42,7 +43,31 @@ const userReqLimiter = rateLimit({
     expiry: 60 * 60 // one hour in seconds
   }),
   max: config.get('rateLimitingUserReqLimit'), // max requests per hour
-  keyGenerator: ipKeyGenerator
+  keyGenerator: ipKeyGenerator,
+  skip: async (req) => {
+    const { libs } = req.app.get('serviceRegistry')
+    let { timestamp, signature } = req.query
+    const path = req.originalUrl
+
+    if (path.includes('/users/clock_status') || path.includes('/users/batch_clock_status')) {
```

Comment:
lets add an early exit here if timestamp and signature don't exist? verifyRequesterIsvalidSP should handle that but if we don't want to take any chances

Reply:
I can see the point in explicitly handling this, but is there a benefit in adding an earlier check? I think logically, this does the same thing?

Reply:
discussed offline -- will add to be extra safe